### PR TITLE
[SPARK-36739][DOCS][PYTHON] Add apache license headers to makefiles

### PR DESCRIPTION
**Pull request by yoda-mon**
### What changes were proposed in this pull request?

Add apache license headers to makefiles of PySpark documents.

### Why are the changes needed?

Makefiles of PySpark documentations do not have apache license headers, while the other files have.

### Does this PR introduce _any_ user-facing change?

No

### How was this patch tested?

`make html` on both Linux and Windows

### CODE REVIEW

File: python/docs/make.bat [hunk 1]
```diff
@@ -1,4 +1,21 @@
-@ECHO OFF
+@echo off
+
```
Comment (yoda-mon) [inline python/docs/make.bat:2]: `spark/bin/pyspark.cmd` uses lower case, so I follow it here.

File: python/docs/make2.bat [hunk 2]
```diff
@@ -1,6 +1,23 @@
-@ECHO OFF
-
-REM Command file for Sphinx documentation
+@echo off
+
```
Comment (yoda-mon) [inline python/docs/make2.bat:2]: `spark/bin/pyspark.cmd` uses lower case, so I follow it here.

**Comment by sarutak** [comment]
ok to test.

**Review by sarutak — APPROVED** [review]

**Comment by sarutak** [comment]
retest this please.

**Comment by sarutak** [comment]
@yoda-mon You changed `bat` files so it's better to ensure that we can successfully build the docs on Windows. Did you confirm that?
If so, could you note in the description that you confirmed `make html` on both Linux and Windows?

**Comment by holdenk** [comment]
Since this is a missing license issue is it a blocker?

**Comment by HyukjinKwon** [comment]
Merged to master and branch-3.2.

**Review by HyukjinKwon — APPROVED** [review]

**Comment by sarutak** [comment]
@holdenk 
> Since this is a missing license issue is it a blocker?

Yeah, I think so.

**Comment by yoda-mon** [comment]
> @yoda-mon You changed `bat` files so it's better to ensure that we can successfully build the docs on Windows. Did you confirm that?
> If so, could you note in the description that you confirmed `make html` on both Linux and Windows?

I did both of them, so I will change the discription like `make html`  on boh Linux and Windows. Thank you.

**Comment by HyukjinKwon** [comment]
Oops, I just noticed that I merged it without seeing https://github.com/apache/spark/pull/33979?fbclid=IwAR3NKZV5NQsSJwXRSwFgJZodP-1uIqF4oKw3L5mxL9Xn7OS7RhfHUKNTAOY#issuecomment-918286059. Good that the comment was addressed after the merge though 👍

**Pull request closed without merging**